eureka/S5_lightcurve_fitting/plots_s5.py

```diff
@@ -76,35 +102,68 @@ def plot_rms(lc, model, meta, fitter):
     History:
     - December 29, 2021 Taylor Bell
         Moved plotting code to a separate function.
+    - January 7-22, 2022 Megan Mansfield
+        Adding ability to do a single shared fit across all channels
     """
     if type(fitter)!=str:
         raise ValueError('Expected type str for fitter, instead received a {}'.format(type(fitter)))
     time = lc.time
-    model_lc = model.eval()
-    residuals = lc.flux - model_lc
-    residuals = residuals[np.argsort(time)]
-
-    rms, stderr, binsz = computeRMS(residuals, binstep=1)
-    normfactor = 1e-6
-    plt.rcParams.update({'legend.fontsize': 11}) # FINDME: this should not be done here but where the rcparams are defined for Eureka
-    plt.figure(int('52{}'.format(str(lc.channel).zfill(len(str(lc.nchannel))))), figsize=(8, 6))
-    plt.clf()
-    plt.suptitle(' Correlated Noise', size=16)
-    plt.loglog(binsz, rms / normfactor, color='black', lw=1.5, label='Fit RMS', zorder=3)  # our noise
-    plt.loglog(binsz, stderr / normfactor, color='red', ls='-', lw=2, label='Std. Err.', zorder=1)  # expected noise
-    plt.xlim(0.95, binsz[-1] * 2)
-    plt.ylim(stderr[-1] / normfactor / 2., stderr[0] / normfactor * 2.)
-    plt.xlabel("Bin Size", fontsize=14)
-    plt.ylabel("RMS (ppm)", fontsize=14)
-    plt.xticks(size=12)
-    plt.yticks(size=12)
-    plt.legend()
-    fname = 'figs/fig52{}_'.format(str(lc.channel).zfill(len(str(lc.nchannel))))+'allanplot_'+fitter+'.png'
-    plt.savefig(meta.outputdir+fname, bbox_inches='tight', dpi=300)
-    if meta.hide_plots:
-        plt.close()
+
+    if lc.share:
```

Comment:
Same comment as above - it should be possible to remove the if/else

Reply:
Done